WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCardCell.swift

```diff
@@ -1,68 +1,36 @@
 import UIKit
 
-class DashboardBlazeCardCell: DashboardCollectionViewCell {
-
-    private var blog: Blog?
-    private weak var presentingViewController: BlogDashboardViewController?
-
-    // MARK: - Views
-
-    private lazy var cardViewModel: BlazeCardViewModel = {
-
-        let onViewTap: () -> Void = { [weak self] in
-            guard let presentingViewController = self?.presentingViewController,
-                  let blog = self?.blog else {
-                return
-            }
-            BlazeEventsTracker.trackEntryPointTapped(for: .dashboardCard)
-            BlazeFlowCoordinator.presentBlaze(in: presentingViewController, source: .dashboardCard, blog: blog)
-        }
-
-        let onEllipsisTap: () -> Void = { [weak self] in
-            BlogDashboardAnalytics.trackContextualMenuAccessed(for: .blaze)
-            BlazeEventsTracker.trackContextualMenuAccessed(for: .dashboardCard)
-        }
+final class DashboardBlazeCardCell: DashboardCollectionViewCell {
+    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?) {
+        BlazeEventsTracker.trackEntryPointDisplayed(for: .dashboardCard)
 
-        let onHideThisTap: UIActionHandler = { [weak self] _ in
-            BlogDashboardAnalytics.trackHideTapped(for: .blaze)
-            BlazeEventsTracker.trackHideThisTapped(for: .dashboardCard)
-            BlazeHelper.hideBlazeCard(for: self?.blog)
+        if RemoteFeature.enabled(.blazeManageCampaigns) {
```

Comment:
[nit] RemoteFeature is an Objective-C bridge for RemoteFeatureFlag.
```suggestion
        if RemoteFeatureFlag.blazeManageCampaigns.enabled() {
```